osu.Game/Graphics/Backgrounds/BeatmapBackgroundWithStoryboard.cs

```diff
@@ -19,16 +22,24 @@ public BeatmapBackgroundWithStoryboard(WorkingBeatmap beatmap, string fallbackTe
         [BackgroundDependencyLoader]
         private void load()
         {
-            LoadComponentAsync(new DrawableStoryboard(Beatmap.Storyboard)
-                {
-                    Clock = new InterpolatingFramedClock(Beatmap.Track),
-                },
-                loaded =>
-                {
-                    AddInternal(loaded);
-                    if (Beatmap.Storyboard.ReplacesBackground)
-                        Sprite.FadeOut(300, Easing.OutQuint);
-                });
+            var storyboard = new Storyboard { BeatmapInfo = Beatmap.BeatmapInfo };
+
+            foreach (var layer in storyboard.Layers)
+            {
+                if (layer.Name != "Fail")
+                    layer.Elements = Beatmap.Storyboard.GetLayer(layer.Name).Elements.Where(e => !(e is StoryboardSampleInfo)).ToList();
+            }
+
+            if (!storyboard.HasDrawable)
+                return;
+
+            if (storyboard.ReplacesBackground)
+            {
+                Sprite.Texture = Texture.WhitePixel;
```

Comment:
Why did this logic change? The old way was fine.

Reply:
When Screen Scaling is either Everything or Excluding Overlays, the scaling container background can be seen through
![image](https://user-images.githubusercontent.com/20495991/120601924-165bc080-c47d-11eb-931d-683eb329680c.png)

Reply:
So you made it black instead? I do not see that as a valid solution.

Reply:
I've checked the behavior on `Player` itself and in the screenshot above is the intended. I'll just fade it out like before.